_ap/18/index.adoc

```diff
@@ -3,76 +3,200 @@ num: 18
+
+Cross-project integrations are facilitated and https://github.com/microsoft/kiota/discussions/2154[healthy conversations] are initiated on both (RH and Microsoft) sides.
+
+==== Better design/architecture
+
+Kiota design is extremely principled and follows https://microsoft.github.io/kiota/extending/[good and detailed guidelines] throughout the project.
+The project leadership shows broad and deep experience in building successful code generators, which are pragmatic and extensible.
+
+==== Better developer experience
+
+The project's architectural design decisions are supported and embraced throughout all of the different implementations so that the user is going to have a consistent and uniform experience while using the generated code across different languages.
+Additionally, the fluent code style of the generated clients provides a better experience for consumers, resulting in higher levels of developer joy.
+
+==== Maintenance
+
+Experience shows that issues are answered within hours, PRs are continuously accepted and merged, and guidance is provided when needed.
+
+Transitive dependencies are kept up to date in a timely manner.
+
+Improvements and bugs are continuously integrated into both the generator and the auxiliary libraries' repositories.
+
+==== Swappable implementation libraries
+
+The design of Kiota allows consumers of the code generator to provide their own https://microsoft.github.io/kiota/extending/corelibrary.html[“opinionated” implementations] of core generated client functionality.
+This is done via a set of interfaces to core functionality that all generated clients leverage (e.g. a Java HTTP client interface).  Default implementations of these interfaces are provided, but alternatives can easily be created and used.
+
+Specifically for the Red Hat use case, we are free to roll out our own implementations of the core libraries to better support Red Hat products and strategy (e.g. Quarkus specific implementations of certain functionality).
+
+Creating alternatives for the provided default libraries is doable in a very reasonable amount of time.
+https://github.com/andreaTP/kiota-utils/blob/1a7dcecd92ce8794fe479e9c37c71d5733a39ff6/http/jdk-http/src/main/java/io/apicurio/kiota/http/JdkHttpRequestAdapter.java#L1[Here] you can find an alternative implementation of the Java HttpClient interface that is based on the client embedded in JDK 11+ (vs. the out-of-the-box implementation that is based on OkHttp).
+We expect to provide:
+
+- one `RHTokenProvider` that will work automatically with [offline tokens](https://access.redhat.com/articles/3626371) and SSO like [we did for Java](https://github.com/andreaTP/kiota-utils/blob/93083650e0374f511abefd099033f05276927deb/rh-auth/src/main/java/com/github/andreatp/kiota/auth/RHAccessTokenProvider.java#L24) for each supported language.
```

Comment:
In the preview these links seems to be malformed (probably you are using markdown syntax instead of adoc)
```suggestion
- one `RHTokenProvider` that will work automatically with https://access.redhat.com/articles/3626371[offline tokens] and SSO like https://github.com/andreaTP/kiota-utils/blob/93083650e0374f511abefd099033f05276927deb/rh-auth/src/main/java/com/github/andreatp/kiota/auth/RHAccessTokenProvider.java#L24[we did for Java] for each supported language.
```

Reply:
thanks!